.github/workflows/test-login.yml

```diff
@@ -36,6 +39,9 @@ jobs:
           echo "server = ${{ steps.oidc.outputs.server }}"
           echo "access_token = ${{ steps.oidc.outputs.access_token }}"
 
+      - name: List environments using CLI (OIDC)
```

Comment:
This test workflow was put in place when the CLI didn't have support for access tokens, it now does so we can improve the testing here.